### PR TITLE
Upgrade CocoaPods to the latest version

### PR DESCRIPTION
**Pull request by fviernau**

### CODE REVIEW

File: .azure-pipelines/LinuxAnalyzerTest.yml [hunk 1]
```diff
@@ -43,11 +43,8 @@ jobs:
       mkdir -p $HOME/go/bin
       curl https://raw.githubusercontent.com/golang/dep/v$GO_DEP_VERSION/install.sh | sh
 
-      # Install 'CocoaPods'. As https://github.com/CocoaPods/CocoaPods/pull/10609 is needed but not yet released.
-      curl -ksSL https://github.com/CocoaPods/CocoaPods/archive/9461b346aeb8cba6df71fd4e71661688138ec21b.tar.gz | tar -zxC .
-      cd CocoaPods-9461b346aeb8cba6df71fd4e71661688138ec21b
-      gem build cocoapods.gemspec
-      sudo gem install cocoapods-1.10.1.gem
+      # Install 'CocoaPods'.
+      gem install cocoapods -v ${COCOAPODS_VERSION}
```
Comment (sschuberth) [inline .azure-pipelines/LinuxAnalyzerTest.yml:47]: Nit: The surrounding curly braces are not required here.

File: Dockerfile [hunk 2]
```diff
@@ -151,14 +152,8 @@ RUN /opt/ort/bin/import_proxy_certs.sh && \
...
-    # Install 'CocoaPods'. As https://github.com/CocoaPods/CocoaPods/pull/10609 is needed but not yet released.
-    curl -ksSL https://github.com/CocoaPods/CocoaPods/archive/9461b346aeb8cba6df71fd4e71661688138ec21b.tar.gz | \
-        tar -zxC . && \
-        (cd CocoaPods-9461b346aeb8cba6df71fd4e71661688138ec21b && \
-            gem build cocoapods.gemspec && \
-            gem install cocoapods-1.10.1.gem \
-        ) && \
-        rm -rf CocoaPods-9461b346aeb8cba6df71fd4e71661688138ec21b && \
+    # Install 'CocoaPods'.
+    gem install cocoapods -v ${COCOAPODS_VERSION} \
```
Comment (sschuberth) [inline Dockerfile:156]: Nit: The surrounding curly braces are not required here.

**Review by sschuberth — DISMISSED** [review]

**Comment by sschuberth** [comment]
[Looks like](https://dev.azure.com/oss-review-toolkit/ort/_build/results?buildId=5494&view=logs&j=9de3b7eb-d67d-5aa9-473b-316991e0154d&t=5103e3ed-6395-56b2-4256-23c91d1f5345&l=2348) you now need to upgrade Ruby, too...

**Comment by sschuberth** [comment]
@fviernau do you still have interest in this, or should I maybe take over?

**Comment by fviernau** [comment]
> @fviernau do you still have interest in this, or should I maybe take over?

Oh this one slipped through somehow, totally forgotten about it. Sure, I'm happy if you take it over.

**Review by sschuberth — APPROVED** [review]

**Pull request merged**